### ISSUE TITLE
爱可可微博热门分享(4.20)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#134]
No 1. <a href="https://weibo.com/1402400261/OaxUVi5hh">用 ORPO 将 Llama 3 的性能提升到新高度</a>
No 2. <a href="https://weibo.com/1402400261/OaxoHm6eo">[CL] Exploring the landscape of large language models: Foundations, ......</a>
No 3. <a href="https://weibo.com/1402400261/Oaz9ih5C7">LLM匿名竞技场的最新结果显示，在English赛道 Llama-3-70b-Instruct 已经超过 Claude ......</a>
No 4. <a href="https://weibo.com/1402400261/OaBLxcg7u">json-repair：提供了一个名为 json-repair 的工具，用于修复从语言模型(LLM)产生的 JSON 格式异常</a>
No 5. <a href="https://weibo.com/1402400261/Oau46tkM3">深度神经网络剪枝相关论文资源列表</a>
No 6. <a href="https://weibo.com/1402400261/Oaz0M09Ku">Penzai：基于 JAX 的神经网络研究工具包，用于构建、编辑和可视化神经网络，可轻松查看模型内部结构，允许在任意地方注入自定义逻辑</a>
No 7. <a href="https://weibo.com/1402400261/OaAdMq4FU">NeRF-Based-SLAM-Incredible-Insights：旨在提供对各种基于 NeRF(神经辐射场)的 Slam(同时定位和建图)算法的全面见解</a>
No 8. <a href="https://weibo.com/1402400261/OaBeEiuFS">General：高性能数据探查器，能使用各种算法在数据中发现许多不同的模式，并允许使用这些算法运行数据清洗场景</a>
No 9. <a href="https://weibo.com/1402400261/Oax2Fsrxi">[LG]《From rr to Q∗Q^*: Your Language Model is Secretly a Q-Function》</a>
No 10. <a href="https://weibo.com/1402400261/OaC28Bhro">BeyondLLM：用于实验、评估和部署基于检索增强生成 (RAG) 的系统的工具包，支持各种大语言模型 (LLM)，旨在减少 LLM 幻觉风险并提高可靠性</a>
No 11. <a href="https://weibo.com/1402400261/OayOl2GT7">LLM API价格比较表</a>
No 12. <a href="https://weibo.com/1402400261/Oau6hxUQx">LibContinual：基于 PyTorch 的开源持续学习工具箱</a>
No 13. <a href="https://weibo.com/1402400261/Oax3tpRlO">[LG]《Learning with 3D rotations, a hitchhiker's guide to SO(3)》</a>
No 14. <a href="https://weibo.com/1402400261/OanTvAiNt">[IR] A Survey on Retrieval-Augmented Text Generation for Large ......</a>
No 15. <a href="https://weibo.com/1402400261/OaC2fz3IW">几篇论文实现代码</a>
No 16. <a href="https://weibo.com/1402400261/OaxvlyGXB">今日推介(第1381期)：语言模型实际上是个Q函数、3D旋转学习指南、单目视频的一致网格重建、视觉基础模型语义分割基准测试方案探索、用微调和迭代推理增强特定领域问答的比较研究 ......</a>
No 17. <a href="https://weibo.com/1402400261/OaEtIkws6">晚安～ [月亮] ​</a>
No 18. <a href="https://weibo.com/1402400261/Oaxrssys1">[LG] Deconstructing Human-AI Collaboration: Agency, Interaction, ......</a>
No 19. <a href="https://weibo.com/1402400261/OaBMw0bqn">Live2D Virtual Human for Chatting based on Unity：寄语Unity的开源聊天虚拟人项目</a>
No 20. <a href="https://weibo.com/1402400261/Oao9Ftpmw">开放语言模型对齐</a>
No 21. <a href="https://weibo.com/1402400261/OaBYufwEK">A Web Component for Math Input：为网页提供易于使用的数学公式输入界面的强大 Web 组件</a>
No 22. <a href="https://weibo.com/1402400261/OaBf60LqQ">BitBLAS：支持混合精度矩阵乘法的库，特别适用于量化 LLM 部署</a>
No 23. <a href="https://weibo.com/1402400261/OawYeov8o">早！[太阳] ​</a>
No 24. <a href="https://weibo.com/1402400261/Oa5M68zO3">MIT新书《计算机视觉基础》，一本关于计算机视觉基础的教材，该教材结合了计算机视觉的经典方法和深度学习的最新进展，并从全面的角度介绍了计算机视觉的基础问题和其与人类感知的关系</a>
No 25. <a href="https://weibo.com/1402400261/OaBYXglHQ">SAE Lens：在语言模型上训练稀疏自编码器</a>
No 26. <a href="https://weibo.com/1402400261/Oau8pBStj">多模态大语言模型(MLLM)幻觉相关论文资源列表</a>
No 27. <a href="https://weibo.com/1402400261/OaBDrrpN6">The largest Awesome List of CLI/TUI programs：最大的精选 CLI/TUI 应用清单，并以 CSV 文件的形式组织了源数据</a>
No 28. <a href="https://weibo.com/1402400261/Oau4NwirE">hf-chat：使用 huggingface/chat-ui 和 huggingface/candle 创建的适用于 macOS 和 iOS 的本地高效的聊天应用程序</a>
No 29. <a href="https://weibo.com/1402400261/Oax2JiZMU">通过在token MDP中重新推导DPO，建立起奖励函数和Q函数的双射关系，从理论和实证两个方面阐明了DPO的工作机制和应用前景。//[LG]《From rr to Q∗Q^*: Your Language Model is Secretly a Q-Function》</a>
No 30. <a href="https://weibo.com/1402400261/OaxaWpNT8">[CV]《How to Benchmark Vision Foundation Models for Semantic Segmentation?》</a>
No 31. <a href="https://weibo.com/1402400261/Oaxh1ryzv">[CL]《Enhancing Q&A with Domain-Specific Fine-Tuning and Iterative Reasoning: A Comparative Study》</a>
No 32. <a href="https://weibo.com/1402400261/OanzLwSVF">[LG]《Many-Shot In-Context Learning》</a>
No 33. <a href="https://weibo.com/1402400261/Oae9YdPBo">[CL]《How faithful are RAG models? Quantifying the tug-of-war between RAG and LLMs' internal prior》</a>
No 34. <a href="https://weibo.com/1402400261/Oax4K9MNa">提出通过 Marching Cubes 算法从关联高斯点云重建质量高且具跨帧对应关系的动态网格。//[CV]《Dynamic Gaussians Mesh: Consistent Mesh Reconstruction from Monocular Videos》</a>
No 35. <a href="https://weibo.com/1402400261/O9S1b0WPy">'qwen-vllm - 通义千问VLLM推理部署DEMO' GitHub: github.com/owenliang/qwen-vllm ......</a>
No 36. <a href="https://weibo.com/1402400261/O9WD6s1R5">视觉语言模型详解</a>
No 37. <a href="https://weibo.com/1402400261/OaBXCyavy">python-barcode：创建标准条形码的简单 Python 库</a>
No 38. <a href="https://weibo.com/1402400261/O9RlP6gHr">SuperMemory：用于构建自己'第二大脑'的工具，可以将你在互联网上收集的信息导入到其中，并且具有类似 ChatGPT 的回答功能。SuperMemory基于 Turborepo 构建，包括三个主要模块：网页 UI、Chrome 扩展和云 AI 后端</a>
No 39. <a href="https://weibo.com/1402400261/OaqEtw0k5">Meta发布Llama 3，新一代开源大型语言模型</a>
No 40. <a href="https://weibo.com/1402400261/OaqFA13o4">非官方的 ComfyUI 魔法变装实现</a>
No 41. <a href="https://weibo.com/1402400261/OaBY1AIPW">CIANNA：专门为天体物理学家设计的深度学习框架，用于天体数据分析，并提供了高级 Python 接口</a>
No 42. <a href="https://weibo.com/1402400261/OaxjOkucv">[CL] Reuse Your Rewards: Reward Model Transfer for Zero-Shot ......</a>
No 43. <a href="https://weibo.com/1402400261/Oaxhfeihj">[IR] Consolidating Ranking and Relevance Predictions of Large ......</a>
No 44. <a href="https://weibo.com/1402400261/OaGEkAj81">存在语料分布不平衡的主语言可以提升稀有语言的表现并增强跨语言泛化，这一效果会随模型和数据规模的扩大而增强。//[CL]《Language Imbalance Can Boost Cross-lingual Generalisation》</a>
No 45. <a href="https://weibo.com/1402400261/OaGKnBhyo">提出基于竞赛编程的USACO基准测试和一系列语言模型推理方法，发现组合自我反思和检索可以大幅提高性能，但仍远未解决该基准测试；进一步的人机交互研究发现目标语言提示可以使GPT-4解决此前无法解决的大多数问题，提供了对语言模型能力和局限的深入洞察。//[CL]《Can Language Models Solve Olympiad Programming?》</a>
No 46. <a href="https://weibo.com/1402400261/OaoBdxMrE">YouTube多语言数据集，包含 YouTube 上的视频信息和相关元数据，总共包含 400k 条记录，字段包括视频标题、链接、描述、发布日期、许可证、原始语言和转录语言等</a>
No 47. <a href="https://weibo.com/1402400261/Oax4EFyrc">[CV]《Dynamic Gaussians Mesh: Consistent Mesh Reconstruction from Monocular Videos》</a>
No 48. <a href="https://weibo.com/1402400261/OaqMQDpHJ">深度立体匹配(Deep Stereo Matching）相关论文资源列表</a>
No 49. <a href="https://weibo.com/1402400261/O9YRebTep">Speechless LLM based Agents：基于LLM 的Agent，具有主动交互、长期记忆、外部工具集成和本地部署能力，旨在建立一个智能协作伙伴，该伙伴可以独立交互、持续发展，并与各种业务场景密切对齐，为企业提供实际价值</a>
No 50. <a href="https://weibo.com/1402400261/IcOM1obJI">'RFSong行人检测网络 - 重新设计的轻量级RFB进行行人检测，AP达到0.7993，参数量仅有3.1MB，200 ......</a>
